Sources/Private/Model/Objects/Transform.swift

```diff
@@ -169,15 +203,21 @@ final class Transform: Codable, DictionaryInitializable {
   /// The positionY of the transform. This is nil if the position property is set.
   let positionY: KeyframeGroup<LottieVector1D>?
 
-  /// The scale of the transform
+  /// The scale of the transform.
   let scale: KeyframeGroup<LottieVector3D>
 
-  /// The rotation of the transform. Note: This is single dimensional rotation.
-  let rotation: KeyframeGroup<LottieVector1D>
+  /// The rotation of the transform on X axis.
+  let rotationX: KeyframeGroup<LottieVector1D>
+
+  /// The rotation of the transform on Y axis.
+  let rotationY: KeyframeGroup<LottieVector1D>
+
+  /// The rotation of the transform on Z axis.
+  let rotationZ: KeyframeGroup<LottieVector1D>
 
   /// The opacity of the transform.
   let opacity: KeyframeGroup<LottieVector1D>
 
-  /// Should always be nil.
-  let rotationZ: KeyframeGroup<LottieVector1D>?
+  /// Rotation. Will always be nil (It's here for `Codable`)
+  let rotation: KeyframeGroup<LottieVector1D>?
```

Comment:
Is there a way for us to remove this property, since it's now unused? Why would we need to keep it around for `Codable` -- could we just keep its coding key `case rotation = "r"` defined above?

Reply:
It's `Codable` screaming the `CodingKeys.rotation` should have a property on the `Transform` class
<img width="488" alt="Screenshot 2023-01-23 at 15 49 17" src="https://user-images.githubusercontent.com/968830/214069458-09076b3f-229d-4c20-a39e-b08369ea0c9c.png">
I think there is a way to trick it by having an enum wrapping the different keys for the `rotationZ`

Something like


```
enum RotationZ: Codable {
   case r(...)
   case rz(...)

   enum CodingKeys: ... {
         case r, rz
   }
} 
```

I can explore this avenue if you want.
I don't really like having a ghost property either

Reply:
Would also be nice to make this property private if possible

Reply:
Private was enough. Good suggestion, was overthinking it ahah